### PR TITLE
[JENKINS-71737] make MesosCloud name uneditable

### PR DESCRIPTION
**Pull request by car-roll**
downstream of https://github.com/jenkinsci/jenkins/pull/8310

Setting `MesosCloud.isNameEditable()` to `false`. This will disable the cloud rename page.

### Testing done

```[tasklist]
### Submitter checklist
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

**Comment by car-roll** [comment]
Does not look like incrementals is working, may need to way for release

File: src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java [hunk 1]
```diff
@@ -145,6 +145,11 @@ public MesosCloud(
         this.master);
   }
 
+  @Override
```
Comment (timja) [inline src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java]: you can just comment this out with a TODO and merge the PR whenever doesn't need to be dependent on the core PR

**Comment by timja** [comment]
> Does not look like incrementals is working, may need to way for release

The Gradle ecosystem only supported incrementals in the last few months and this plugin hasn't been updated for a couple of years

File: src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java (hunk 1 above)
Comment (car-roll) [reply src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java]: Commented out with a TODO in 73d45ab1d7e3c03ab931a02033b4f7415734e14c

**Comment by car-roll** [comment]
@timja, do you have merge permissions to this repository? I do not

**Comment by timja** [comment]
> @timja, do you have merge permissions to this repository? I do not

No I am not a maintainer here

**Comment by car-roll** [comment]
cc @jeschkies for review

**Comment by car-roll** [comment]
moving to draft and waiting for a non-incremental version to bump this PR to

**Comment by jeschkies** [comment]
@car-roll I'm trying to upgrade Gradle but it's been a while that I've used Java https://github.com/jenkinsci/mesos-plugin/pull/421

**Pull request closed without merging**